tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+import wazuh_testing.fim as fim
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+
+
+# Helper functions
+
+def extra_configuration_after_yield():
+    fim.delete_registry(fim.registry_parser[key], sub_key_2, fim.KEY_WOW64_64KEY)
+
+
+def check_event_type_and_path(fim_event, monitorized_registry):
+    check_event = False
+    if fim_event['type'] == 'added':
+        registry_event_path = fim_event['path']
+        if monitorized_registry.lower() == registry_event_path.lower():
+            check_event = True
+
+    return check_event
+
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
```

Comment:
```suggestion
key = 'HKEY_LOCAL_MACHINE'
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/c89eb367365e53d034a3f04058598377b3acc2c6)